docs/src/dev_guide/time_series.md

```diff
@@ -38,3 +38,63 @@ deserialize the system.
 1. Add an instance of `InfrastructureSystems.TimeSeriesContainer` to the component struct.
 2. Implement the method `InfrastructureSystems.get_time_series_container` for the
    component. It must return the TimeSeriesContainer object.
+
+
+## Data Format
+Time series arrays are stored in an HDF5 file accoring the format described here.
+
+The root path ``/time_series`` defines these HDF5 attributes to control deserialization:
+
+- ``data_format_version``: Designates the InfrastructureSystems format for the file.
+- ``compression_enabled``: Specifies whether compression is enabled and will be used for new time series.
+- ``compression_type``: Specifies the type of compression being used.
+- ``compression_level``: Specifies the level of compression being used.
+- ``compression_shuffle``: Specifies whether the shuffle filter is being used.
+
+Each time series array is stored in an HDF5 group named with the array's UUID.
+Each group contains a dataset called ``data`` which contains the actual data.
+Each group also contains a group called ``component_references`` which contains
+an HDF5 attribute for each component reference. The component reference uses the
+format ``<component_uuid>__<time_series_name>``.
```

Comment:
label?

Reply:
`label` isn't used in the code; `name` is.